README.md

```diff
@@ -145,7 +145,7 @@ In order to run competition matches you'll need to:
    Note: you can change the version of Python which Webots uses from the UI --
    go to "Tools" > "Preferences" > "General" > "Python command".
 
-   We are using Python 3.7, though it shouldn't matter whether it's a system
+   We are using Python 3.8, though it shouldn't matter whether it's a system
```

Comment:
What does this paragraph mean bereft of the virtual competition context?

Reply:
Not much, though I don't think this is the PR to address that.